### PR TITLE
Pioneer 18 corrections

### PR DESCRIPTION
**Pull request by cdobs**
Correcting deployment times for 2 Pioneer 18 glider deployments

### CODE REVIEW

File: deployment/CP05MOAS-GL380_Deploy.csv [hunk 1]
```diff
@@ -65,9 +65,9 @@ TS20211214,,TS20220221,,CP05MOAS-GL380-02-FLORTM000,11,1,2021-12-15T07:36:00,202
...
\ No newline at end of file
+AR66,,TS20220714,,CP05MOAS-GL380-00-ENG000000,12,1,2022-04-10T20:20:00,2022-07-14T17:06:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGCON-GLDRCP-00380,39.93425,-70.58905,,1000,1000,FZ-1,,
+AR66,,TS20220714,,CP05MOAS-GL380-01-ADCPAM000,12,1,2022-04-10T20:20:00,2022-07-14T17:06:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGINS-ADCPAM-50867,39.93425,-70.58905,,1000,1000,FZ-1,,
+AR66,,TS20220714,,CP05MOAS-GL380-02-FLORTM000,12,1,2022-04-10T20:20:00,2022-07-14T17:06:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGINS-FLORTM-04251,39.93425,-70.58905,,1000,1000,FZ-1,,
+AR66,,TS20220714,,CP05MOAS-GL380-03-CTDGVM000,12,1,2022-04-10T20:20:00,2022-07-14T17:06:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGINS-CTDGVM-09037,39.93425,-70.58905,,1000,1000,FZ-1,,
+AR66,,TS20220714,,CP05MOAS-GL380-04-DOSTAM000,12,1,2022-04-10T20:20:00,2022-07-14T17:06:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGINS-DOSTAM-00171,39.93425,-70.58905,,1000,1000,FZ-1,,
+AR66,,TS20220714,,CP05MOAS-GL380-05-PARADM000,12,1,2022-04-10T20:20:00,2022-07-14T17:06:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGINS-PARADM-50152,39.93425,-70.58905,,1000,1000,FZ-1,,
```
Comment (snwhite) [inline deployment/CP05MOAS-GL380_Deploy.csv:73]: Looks good.  UTC time matches QLR.

File: deployment/CP05MOAS-GL388_Deploy.csv [hunk 2]
```diff
@@ -71,12 +71,12 @@ TS20220104,,TS20220221,,CP05MOAS-GL388-02-FLORTM000,12,1,2022-01-04T20:57:00,202
...
+AR66,,AR66,,CP05MOAS-GL388-00-ENG000000,13,1,2022-04-10T20:00:00,2022-04-10T22:54:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGCON-GLDRCP-00388,39.93303,-70.58503,,1000,1000,FZ-1,,
+AR66,,AR66,,CP05MOAS-GL388-01-ADCPAM000,13,1,2022-04-10T20:00:00,2022-04-10T22:54:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-ADCPAM-43106,39.93303,-70.58503,,1000,1000,FZ-1,,
+AR66,,AR66,,CP05MOAS-GL388-02-FLORTM000,13,1,2022-04-10T20:00:00,2022-04-10T22:54:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-FLORTM-04467,39.93303,-70.58503,,1000,1000,FZ-1,,
+AR66,,AR66,,CP05MOAS-GL388-03-CTDGVM000,13,1,2022-04-10T20:00:00,2022-04-10T22:54:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-CTDGVM-09363,39.93303,-70.58503,,1000,1000,FZ-1,,
+AR66,,AR66,,CP05MOAS-GL388-04-DOSTAM000,13,1,2022-04-10T20:00:00,2022-04-10T22:54:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-DOSTAM-00418,39.93303,-70.58503,,1000,1000,FZ-1,,
+AR66,,AR66,,CP05MOAS-GL388-05-PARADM000,13,1,2022-04-10T20:00:00,2022-04-10T22:54:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-PARADM-50238,39.93303,-70.58503,,1000,1000,FZ-1,,
```
Comment (snwhite) [inline deployment/CP05MOAS-GL388_Deploy.csv:79]: Looks good.  UTC time matches QLR.

**Review by snwhite — APPROVED** [review]

File: deployment/CP05MOAS-GL380_Deploy.csv (hunk 1 above)
Comment (rtravis-whoi) [inline deployment/CP05MOAS-GL380_Deploy.csv:73]: looks good

File: deployment/CP05MOAS-GL388_Deploy.csv (hunk 2 above)
Comment (rtravis-whoi) [reply deployment/CP05MOAS-GL388_Deploy.csv:79]: looks good

**Review by rtravis-whoi — APPROVED** [review]

**Pull request merged**